### PR TITLE
Convert more HA tests to YAML scheduling

### PR DESCRIPTION
**Pull request by juadk**
This PR adds more HA tests fully converted to YAML.

* Conversion for hawk, pacemaker-cts, zalpha and rolling upgrade migration.

- [Job group configuration Merge Request](https://gitlab.suse.de/qa-css/openqa_ha_sap/-/merge_requests/260)
- Related ticket: https://jira.suse.com/browse/TEAM-3572
- Needles: N/A
- Verification run: 
**x86_64:**
Rolling upgrade: [node1](http://1b143.qa.suse.de/tests/7442) [node2](http://1b143.qa.suse.de/tests/7443)
Pacemaker cts: [node1](http://1b143.qa.suse.de/tests/7439) [node2](http://1b143.qa.suse.de/tests/7440) [client](http://1b143.qa.suse.de/tests/7438)
Hawk - Haproxy: [node1](http://1b143.qa.suse.de/tests/7424) [node2](http://1b143.qa.suse.de/tests/7425) [client](http://1b143.qa.suse.de/tests/7423)
**aarch64:**
Pacemaker cts: [node1](http://1b143.qa.suse.de/tests/7428) [node2](http://1b143.qa.suse.de/tests/7429) [client](http://1b143.qa.suse.de/tests/7427)

s390x zalpha cluster is configured on the dev group first.

### CODE REVIEW

**Review by ricardobranco777 — APPROVED** [review]
LGTM
Thx!

File: schedule/ha/bv/s390x_cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  to different hostnames for each node. All jobs with the exception of the
+  parent job must include a PARALLEL_WITH setting referencing the parent job.
+  HA_UNICAST must be set to 1 because we can not configure multicast corosync on
+  s390x. ISCSI_LUN_INDEX must be set to an available LUN Id starting point,  HA
+  tests will take 5 consecutive LUNs; 1 for SBD, 2 for cluster_md and 2 for
+  drbd_passive. Be very careful with that settings because you may erase LUNs
+  used by other tests. More information here,
+  https://gitlab.suse.de/hsehic/qa-css-docs/-/blob/master/ha/openqa.md#information-specific-to-s390x-tests
+  ISCSI_SERVER must be set to the iscsi server ip address. And of course,
+  NICTYPE and WORKER_CLASS must be set to 'tap' in the job group directly in qemu based jobs.
```
Comment (alvarocarvajald) [inline schedule/ha/bv/s390x_cluster.yaml:30]: `NICTYPE=tap` and `WORKER_CLASS=tap` settings are not needed in `svirt`

File: schedule/ha/bv/s390x_cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  YAML_SCHEDULE must point to this file.
+
+  Below are the optional settings.
+
+  HA_CLUSTER_DRBD can be defined for enabling DRBD test. HA_REMOVE_NODE can be
+  defined for testing a node removal. USE_SYSRQ_FENCING can be defined for
+  fencing through sysrq instead of 'crm node fence' command.
```
Comment (alvarocarvajald) [inline schedule/ha/bv/s390x_cluster.yaml:37]: Excellent! We could add zbeta with these settings.

**Review by alvarocarvajald — APPROVED** [review]
I have some small comments with regards to the s390x schedule, but otherwise this LGTM.

File: schedule/ha/bv/s390x_cluster.yaml (hunk 1 above)
Comment (juadk) [reply schedule/ha/bv/s390x_cluster.yaml:30]: Good point thx

**Review by ldevulder — APPROVED** [review]
LGTM

**Pull request merged**